code/unit_tests/map_tests.dm

```diff
@@ -47,7 +47,7 @@ datum/unit_test/apc_area_test/start_test()
 		,/area/medical/genetics
 	)
 
-	for(var/area/A in world)
```

Comment:
Actually, this is a unit_test. Can you leave the in world present here? It's more robust that way.